samples/highcharts/data-tools/datacursor-html/demo.css

```diff
@@ -0,0 +1,37 @@
+#wrapper {
+    display: flex;
+    justify-content: stretch;
+}
+
+#container1,
+#container2 {
+    flex-grow: 1;
+    text-align: center;
+}
+
+table {
+    min-width: 100%;
+}
+
+table td,
+table th {
+    line-height: 1.5em;
+    padding: 0.25em 0.5em;
+    vertical-align: top;
+}
+
+table thead th {
+    background-color: #ddd;
+}
+
+table tbody td {
+    background-color: #eee;
+}
+
+tr.Focus td {
```

Comment:
I have a problem with the class name because of two reasons
- capitalic at the first letter
- use focus which is almost the same like `:focus` selector.

My suggestion is to use another name such as `current`.